### PR TITLE
Add compat data for ::slotted pseudo-element

### PR DESCRIPTION
**Pull request by chrisdavidmills**

### CODE REVIEW

**Comment by teoli2003** [comment]
This looks like a recent addition. @chrisdavidmills: Do you have a bug number so I can check the content of the table, and if is behind a flag?

**Comment by chrisdavidmills** [comment]
This is a good point — serves me right for doing a PR so late at night ;-)

The ::slotted() meta bug seems to be this one: https://bugzilla.mozilla.org/show_bug.cgi?id=1424604

And this one suggests that it'll only be parsed if the webcomponents pref is on: https://bugzilla.mozilla.org/show_bug.cgi?id=1425834

Ummm, there are three ;-)

No matter. I have tested my demo, and it only parses ::slotted if dom.webcomponents.customelements.enabled and dom.webcomponents.shadowdom.enabled are set to true.

So I guess I need to add some notes to Firefox/FxA.

I am not sure about Chrome/Opera versions. I had difficulty digging up any more exact bugs, but I know that it works in both Chrome 63 and Opera 50 (release versions).

**Comment by teoli2003** [comment]
Thanks, Chris!

@jpmedley  may help us with Chromia :-)

**Comment by teoli2003** [comment]
Also, I wonder if it does anything that is actually usable on Firefox right now, or if it only an implementation in progress, and we should keep at false (not implemented) until the implementation is finished. @emilio, as the implementer, what do you think?

**Comment by jpmedley** [comment]
Chrome 50/Opera 37

**Comment by chrisdavidmills** [comment]
Thanks folks. I've updated the info with the new details.

**Comment by emilio** [comment]
@chrisdavidmills it should be usable on nightly, but it's preffed-off (so not shipped). So given it's not usable on release I'd maybe punt on it.

File: css/selectors/slotted.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+              "version_added": "50"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true,
```
Comment (teoli2003) [inline css/selectors/slotted.json]: "59"

File: css/selectors/slotted.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                },
+                {
+                  "type": "preference",
+                  "name": "dom.webcomponents.shadowdom.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": {
+              "version_added": true,
```
Comment (teoli2003) [inline css/selectors/slotted.json]: "59"

**Comment by chrisdavidmills** [comment]
I've updated the data as requested; I've also added an entry for it to the experimental features page:

https://developer.mozilla.org/en-US/Firefox/Experimental_features#CSS

@emilio I think we should include it now; quite a few devs are interested in how they can enable and play with new features, even if they aren't useful for users yet.

**Comment by emilio** [comment]
Ok, sounds good then! :)

**Pull request merged**